response_operations_ui/assets/scss/includes/_overrides.scss

```diff
@@ -187,3 +187,19 @@ button.hidden {
 .error-message {
     color: #d0021b;
 }
+
+.ons-tabs__list {
```

Comment:
This was incorrectly implemented in this PR https://github.com/ONSdigital/response-operations-ui/pull/882. The CSS was put into the auto generated code, so would be overwritten if anyone called make minify. This is the source file, the PR put it in the destination file